inference-engine/src/mkldnn_plugin/CMakeLists.txt

```diff
@@ -42,7 +42,8 @@ endif()
 target_link_libraries(${TARGET_NAME} PRIVATE mkldnn
                                              inference_engine
                                              inference_engine_transformations
-                                             inference_engine_lp_transformations)
+                                             inference_engine_lp_transformations
+                                             ngraph::shape_inference)
```

Comment:
Why do you need to link MKLDNN plugin with this library? I don't see that you are using anything from the library.

Reply:
yes, I will introduce a library in CPU plugin that will use shape inferences. stay tuned :)